iroh-net/src/magicsock/endpoint.rs

```diff
@@ -1312,13 +1244,6 @@ pub struct EndpointInfo {
     pub latency: Option<Duration>,
 }
 
-#[derive(Default, Debug, PartialEq, Eq, Clone, Copy, Hash)]
-enum Index {
```

Comment:
Thanks for removing this! I was reading in the code and wondering that it was never used apart from a deletion flag.